media/js/src/simulations/simulation4/whatAreLogarithms.jsx

```diff
@@ -32,6 +33,53 @@ export const WhatAreLogarithmRegs = ({
             .trim()
             .toLowerCase();
 
+    const getModelFormula = (model) => {
+        switch (model) {
+        case 'logLinear':
+            return '\\ln(y) = \\widehat{\\beta}_0 + \\widehat{\\beta}_1 x';
+        case 'linearLog':
+            return (
+                '\\hat{y} = \\widehat{\\beta}_0 + ' +
+                '\\widehat{\\beta}_1 \\ln(x)'
+            );
+        case 'logLog':
+            return (
+                '\\widehat{\\ln(y)} = \\widehat{\\beta}_0 + ' +
+                '\\widehat{\\beta}_1 \\ln(x)'
+            );
+        default:
+            return '';
+        }
+    };
+
+    const getRegFormula = (model, fit) => {
+        const formulas = {
+            logLinear: {
+                linearFit: '\\hat{y} = -669.23 + 224.14x',
+                logLinearFit: '\\widehat{\\ln(y)} = 1.05 + 0.97x'
+            },
+            linearLog: {
+                linearFit: '\\hat{y} = 5.57 + 0.0025x',
+                linearLogFit: '\\hat{y} = 1.03 + 0.98\\ln(x)'
+            },
+            logLog: {
+                linearFit: '\\widehat{\\nl(y)} = 106.15 + 2.48\\ln(x)',
```

Comment:
Sorry, I made this one wrong because this is linear-linear.
`\\hat{y} = 106.15 + 2.48x`